src/lib/style/components/_project-view-card.scss

```diff
@@ -10,7 +10,15 @@
   text-align: left;
 
   img {
+    display: block;
     max-width: 100%;
+    max-height: 30%;
+    width: auto;
+    height: auto;
+
+    // Center the image
+    margin-left: auto;
+    margin-right: auto;
```

Comment:
This solution to use margin to center the image is interesting. I normally would apply `text-align: center` on the image's parent div to center the image, providing that the image is the only element within the parent div. I have to try your solution sometimes.

Reply:
I have other elements in the div that utilize the parent's `text-align: left`